### ISSUE TITLE
Create POST APIs

### ISSUE THREAD

**Issue by aviaryan** [fossasia/open-event-orga-server#442]
See https://github.com/fossasia/open-event-orga-server/issues/336 for the schema

Currently these POST methods will be publicy accessible (login needed). Only after #429 is done (which in on hold currently), the access control can be added to it.

**Comment by rafalkowalski** [comment]
@aviaryan please work on gsoc2016 branch i've updated master to gsoc

**Comment by aviaryan** [comment]
@juslee Will we provide the feature to create an event from the REST API ?

@rafalkowalski Are you sure it will be a good idea considering the gsoc branch tests are failing ?

**Comment by juslee** [comment]
@aviaryan every feature you think should be there. You want the REST APIs to be as comprehensive as possible so that your web-app and orga-server can eventually just call the APIs.

**Comment by aviaryan** [comment]
@juslee Ok. Will keep that in mind. 

And as for my question, I will add the method to create event in the API.
